lib/ansible/modules/packaging/os/flatpak_remote.py

```diff
@@ -168,7 +168,7 @@ def remote_exists(module, binary, name, method):
     # The query operation for the remote needs to be run even in check mode
     output = _flatpak_command(module, False, command)
     for line in output.splitlines():
-        listed_remote = line.split()
+        listed_remote = line.split('\t')
```

Comment:
As asked by @gundalow, to avoid breaking compatibility with old releases, you should probably split on either space or tabs.

Reply:
@bmalynovytch Wouldn't the simple approach of supporting tabs and spaces potentially lead to more breakage in future when flatpak again decides to change their output format in some subtle way? I mean the reason why I implemented it this way originally was only because there was no better way to get output from the `flatpak` binary.
From version 1.2 (the one that breaks with the current code), however, there now is a parameter called `--columns` that allows us to control the output of the `list-remote` command. That way we should be more future-proof by specifying `--columns=name` in case of flatpak >= 1.2.
I implemented this behaviour for the `flatpak` module in PR #51482 (still unmerged). Please have a look and comment there if you have any views. I'm not religious about my solution. If you guys think it's too complex, we could go for a more naive approach. Maybe it's not worth it.
But what I do feel strongly about is to implement the behaviour the same way among the `flatpak` and the `flatpak_remote` modules, for consistency.

Reply:
In fact, the good question is: with your contrib, do older versions still work or will it break compatibility ? (ie. Ansible 2.8 + Flatpak < 1.2.0)
If so, we need to be able to handle the situation (like you did in #51482)

Reply:
I ran the integration tests on this PR and they confirm that the code above works for both flatpak 1.2.0 **and** before.
I mean, I like the fact that @JayKayy's solution is super simple.
Even if it is possible that flatpak breaks it's output in future again, we don't know if they actually will. So maybe one could regard a solution like mine from #51482 as premature optimisation...